packages/lesswrong/components/comments/CommentsTimelineSection.tsx

```diff
@@ -87,7 +87,7 @@ const CommentsTimelineSection = ({
     >
       <Components.CommentsTimeline
         treeOptions={{
-          post: post,
+          refetch,
```

Comment:
I've had to add this manual refetch for when you post a comment (so your own comment shows up), this was happening automatically before but some change I made stopped it from working. If anyone knows how a refetch gets triggered automatically on e.g. the posts page please lmk because I would like to remove this